Samples/WindowsML/cpp-cmake/CMakeLists.txt

```diff
@@ -0,0 +1,70 @@
+#----------------------------------------------------------------------------------------------------------------------
+#
+#----------------------------------------------------------------------------------------------------------------------
+cmake_minimum_required(VERSION 3.31)
+
+cmake_policy(SET CMP0141 NEW) # MSVC debug information format flags are selected by an abstraction.
+cmake_policy(SET CMP0117 NEW) # MSVC RTTI flag /GR is not added to CMAKE_CXX_FLAGS by default.
+
+if(CMAKE_GENERATOR MATCHES "^Visual Studio")
+    # To prevent builds (including try_compile) from picking up top-level 'Directory.Build.*' files,
+    # write versions specific to the CMake builds at the root of the CMake build.
+    file(WRITE "${CMAKE_BINARY_DIR}/Directory.Build.rsp" "-m -graphBuild:True -p:UseMultiToolTask=True")
+    file(WRITE "${CMAKE_BINARY_DIR}/Directory.Build.props" "<Project></Project>")
+    file(WRITE "${CMAKE_BINARY_DIR}/Directory.Build.targets" "<Project></Project>")
+elseif(CMAKE_GENERATOR MATCHES "^Ninja")
+    if(NOT (DEFINED ENV{Platform}))
+        message(FATAL_ERROR "When using the Ninja generator, you must build from a platform-specific Developer Commnand Prompt.")
+    endif()
+endif()
+
+project(WinMLSamples LANGUAGES CXX)
+
+set(CMAKE_CXX_STANDARD_LIBRARIES "")
+set(CMAKE_C_STANDARD_LIBRARIES "")
+set(CMAKE_MSVC_DEBUG_INFORMATION_FORMAT Embedded)
+
+include(FetchContent)
+
+FetchContent_Declare(
+  CMakeNuGetPackage
+  GIT_REPOSITORY https://github.com/mschofie/NuGetCMakePackage
+  GIT_TAG 74545405a30aa3e358665ea1431948f61875edad
+)
+
+FetchContent_MakeAvailable(CMakeNuGetPackage)
+
+add_nuget_library(Microsoft.Windows.ImplementationLibrary 1.0.250325.1)
```

Comment:
Not sure it matters, but I get `'Microsoft.Windows.ImplementationLibrary 1.0.250325.1' is not found in the (upstream feeds)`

Reply:
Hmm... I must've had packages in my machine cache. I'll double check versions to make sure I'm using public things.

TBH, I should probably have a convention where versions are expressed as variables in a single file; like `Directory.Packages.props`.

Reply:
Thanks! It looks like the NuGet feed that the samples pull from doesn't have the latest 'wil' or 'CppWinRT'. I've snapped back to the latest available on the feed.